src/dns/aardvark.rs

```diff
@@ -265,14 +268,27 @@ impl Aardvark {
                                                 container_ip_v6 = container_ip.to_string();
                                             }
 
+                                            // unwrap is safe since we already have statusblock
+                                            // for this network
+                                            let mut name_vector =
+                                                Vec::from([container_name.clone()]);
+                                            if let Some(alias) = &per_network_opts
+                                                .get(&network_name)
```

Comment:
remind me, what happens with this call if there is NO aliases key?

Reply:
@baude if key is not defined `if` would not satisfy so nothing would happen.

Btw I also test this after: https://github.com/containers/aardvark-dns/pull/34 and https://github.com/containers/netavark/pull/188 it is working as expected.